iohub/ngff.py

```diff
@@ -340,7 +341,19 @@ def downscale(self):
         raise NotImplementedError
 
     def tensorstore(self):
-        raise NotImplementedError
+        import tensorstore as ts
+
+        assert self.path != "", "Zarr array must be stored on a file system."
```

Comment:
In theory an iohub NGFF store is always on a local FS, so this might be redundant.